locations/spiders/bnp_paribas_bank_pl.py

```diff
@@ -0,0 +1,58 @@
+import scrapy
+from chompjs import chompjs
+
+from locations.categories import Categories, Extras, apply_category, apply_yes_no
+from locations.dict_parser import DictParser
+from locations.hours import DAYS, OpeningHours
+
+BRANDS = {
+    "BNP Paribas Bank Polska": ("BNP Paribas Bank Polska", "Q20744004"),
+    "Euronet": ("Euronet", "Q5412010"),
+    "Planet Cash": ("Planet Cash", "Q117744569"),
+}
+
+
+class BNPParibasBankPLSpider(scrapy.Spider):
+    name = "bnp_paribas_bank_pl"
+    start_urls = ["https://www.bnpparibas.pl/_js_places/time20230712142358/places.js"]
```

Comment:
Since this has a timestamp on it, can we extract the URL from some page so that we don't have to update it in the future?

Reply:
Nice catch, thanks!